app/imports/client/react/components/subcards/FilesField.js

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Field } from 'react-final-form';
+import { getValue } from 'plio-util';
+import { append, equals, reject, eqBy } from 'ramda';
+
+import { renderComponent } from '../../helpers';
+
+const FilesField = ({ onChange, ...props }) => (
+  <Field
+    subscription={{ value: true }}
+    isEqual={eqBy(getValue)}
```

Comment:
The function you're using expects an array of objects with the key `value`. This field plain array of ids.
```suggestion
    isEqual={equals}
```